src/parsing/parsable.rs

```diff
@@ -172,8 +172,10 @@ impl sealed::Sealed for Rfc3339 {
         };
 
         // The RFC explicitly allows leap seconds. We don't currently support them, so treat it as
-        // the previous moment.
+        // the nearest preceding moment that can be represented, but set a flag to perform
+        // special validation when the time value is constructed.
         if parsed.second == Some(60) {
+            parsed.leap_second_input = true;
```

Comment:
In my opinion a better approach would be simply set the second to 60 unconditionally and only check whether it's a leap second in the `TryFrom` impl. Then the only thing that would be necessary is knowing whether the format is well-known (which can be set unconditionally).

I am aware that this is currently permitted on the latest release, but this could certainly be considered a bug fix due to the lack of information in the final value. It's a mistake on my end, admittedly.

Reply:
OK, will redo as proposed.